eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -53,6 +53,13 @@ services:
         factory_method: buildStorageEngine
         public: false
 
+    ezpublish.spi.search_handler:
+        alias: ezpublish.spi.search_engine
```

Comment:
Yes, I understand that. Same as `ezpublish.spi.storage_engine` is private BTW.
But let's try to get at this again when https://github.com/ezsystems/ezpublish-kernel/pull/1167 is rebased on master.

Thanks for a quick response.